File: python/examples/get_player_team.py
import os
import sys

from mythicmc import DEFAULT_BASE_URL, MythicMC

key = os.environ["MYTHICMC_API_KEY"]
base_url = os.environ.get("MYTHICMC_API_URL", DEFAULT_BASE_URL)
player = sys.argv[1] if len(sys.argv) > 1 else "Vicente_1313"

with MythicMC(key, base_url=base_url) as api:
    reply = api.get_player_team(player)

team = reply.team
if team is None:
    print(f"{reply.name} is not in a team")
else:
    print(f"{reply.name} is in [{team.prefix}] {team.name}, team level {team.level}, role {team.role or 'unknown'}")

File: typescript/examples/get-player-team.ts
import { MythicMC } from '../src/index.ts'

const api = new MythicMC({ apiKey: process.env.MYTHICMC_API_KEY!, baseUrl: process.env.MYTHICMC_API_URL })
const player = process.argv[2] ?? 'Vicente_1313'

const { name, team } = await api.getPlayerTeam(player)

if (!team) {
  console.log(`${name} is not in a team`)
} else {
  console.log(`${name} is in [${team.prefix}] ${team.name}, team level ${team.level}, role ${team.role ?? 'unknown'}`)
}
